resources/views/user_list.blade.php

```diff
@@ -1 +1,9 @@
-<div class = "container">入門課題2です</div>
\ No newline at end of file
+@foreach($users as $user)
+        
+       id: {{$user->id}}<br>
+       user_id: {{$user->user_id}}<br>
+       password: {{$user->password}}<br>
+       =========================<br>
+   
+        
+    @endforeach
```

Comment:
細かいところですが、現場レベルだとインデントをきちんと揃えるなどはソースの可読性の観点から必須のところなので、勉強の段階から意識づけを行っておいた方がいいですね。
```suggestion
@foreach($users as $user)
        
       id: {{$user->id}}<br>
       user_id: {{$user->user_id}}<br>
       password: {{$user->password}}<br>
       =========================<br>
   
        
@endforeach
```

Reply:
ご指摘ありがとうございます。
こちら修正したファイルを再コミットいたしました。